EXILED/Exiled.API/Features/Toys/Speaker.cs

```diff
@@ -87,12 +87,25 @@ public float MinDistance
         }
 
         /// <summary>
-        /// Creates a <see cref="Speaker"/>.
+        /// Creates a new <see cref="Speaker"/>.
         /// </summary>
-        /// <returns>The <see cref="Speaker"/>.</returns>
-        public static Speaker Create()
+        /// <param name="position">The position of the <see cref="Speaker"/>.</param>
+        /// <param name="rotation">The rotation of the <see cref="Speaker"/>.</param>
+        /// <param name="scale">The scale of the <see cref="Speaker"/>.</param>
+        /// <param name="spawn">Whether the <see cref="Speaker"/> should be initially spawned.</param>
+        /// <returns>The new <see cref="Speaker"/>.</returns>
+        public static Speaker Create(Vector3? position, Vector3? rotation, Vector3? scale, bool spawn)
```

Comment:
if you consider them as nullable maybe add a `null` default value?

Reply:
we can add default value yeah, i just copied the method from another toy